conformance/fhir-ig-davinci-epdx/src/main/java/com/ibm/fhir/ig/davinci/pdex/EPDXResourceProvider.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.ig.davinci.pdex;
+
+import com.ibm.fhir.registry.util.PackageRegistryResourceProvider;
+
+public class EPDXResourceProvider extends PackageRegistryResourceProvider {
```

Comment:
Is there a reason why we went with EPDX as a prefix here and we use `pdex` as a suffix and package name in other places? Should this be `PDEXResourceProvider`?

Reply:
I've updated the class name to use PDEX (it's very confusing on their site).

Reply:
I agree its confusing.  My "best guess" is that originally they used `epdx` and so thats what is in the repo and the path of the IG on the build site.
However, all the canonical URLs in the IG use `pdex` and not `epdx` and I think that signals that this will be published at `http://hl7.org/fhir/us/davinci-pdex` (rather than `http://hl7.org/fhir/us/davinci-epdx`)...therefor I'm thinking that we should use `pdex` everywhere and squash all usage of `epdx` (include the module name).

Reply:
Changed to PDEX